src/Crypto/ECC/Ed25519/Sign.hs

```diff
@@ -44,7 +44,7 @@ import safe Prelude ((==),($),(<),IO,return,pure,Either(Left,Right),String,(&&))
 import safe qualified Crypto.Fi as FP
 import safe qualified Data.ByteString as BS
 #ifndef mingw32_HOST_OS
-import safe qualified Data.ByteString.Lazy.Char8 as BS8
```

Comment:
Unfortunately, `liquid-bytestring` doesn't export Data.ByteString.Lazy.Char8 yet. I will see if I can do something about it at least in my fork.